modules/phase_field/include/userobjects/ComputeGrainForceAndTorque.h

```diff
@@ -0,0 +1,71 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+
+#ifndef COMPUTEGRAINFORCEANDTORQUE_H
+#define COMPUTEGRAINFORCEANDTORQUE_H
+
+#include "GrainForceAndTorqueInterface.h"
+#include "ElementUserObject.h"
+#include "ComputeGrainCenterUserObject.h"
```

Comment:
You can forward-declare this class using
```
class ComputeGrainCenterUserObject;
```
and move the include into the.C file.